src/Build.OM.UnitTests/Definition/ProjectItem_Tests.cs

```diff
@@ -806,6 +808,9 @@ public void ProjectGetterResultsInDriveEnumerationException(string unevaluatedIn
         [InlineData(@"z:\**\*.cs")]
         public void ProjectGetterResultsInWindowsDriveEnumerationWarning(string unevaluatedInclude)
         {
+            // let's create the mapped drive only once it's needed by any test, then let's reuse;
+            _mappedDrive ??= new DummyMappedDrive();
+            unevaluatedInclude = UpdatePathToMappedDrive(unevaluatedInclude, _mappedDrive.MappedDriveLetter);
```

Comment:
nit: just wondering if we can request the specific drive instead, imo this makes it a bit more readable

```
using DummyMappedDrive mappedDrive = DriveMapping.GetDrive("z");

// test content


```

Reply:
The drive to request is unknown upfront - it needs to be untaken on the system.
So the `_mappedDrive` takes care about determining one that is free and using it.
Added `DriveMapping` is utility class that can be used for that purpose.

tl;dr;: Explicit assigning is intentionally hidden in this wrapper.

Reply:
in this case, using placeholder in test data might be more readable, and then it is replaced by `DriveMapping`.
e.g: `"%DRIVE%:\**\*.cs"`

imo, for test purposes picking up 'z' or other letter at the end of abc should work.

Reply:
I like the `%DRIVE%:` suggestion! - updated.

I wouldn't use any fixed drive letter though (ppl tend to map shares to various  drive letters). The dynamic drive letter allocation logic isn't complicated - so 'd keep it.